apps/heft/src/plugins/Webpack/WebpackPlugin.ts

```diff
@@ -68,9 +68,13 @@ export class WebpackPlugin implements IHeftPlugin {
         port: 8080
       };
 
+      // Require webpack-dev-server here because it sets the "WEBPACK_DEV_SERVER" env
+      // variable when it's loaded, which can cause problems when webpack isn't run
+      // in serve mode.
+      const WebpackDevServer: typeof TWebpackDevServer = require('webpack-dev-server');
```

Comment:
Consider using `Import.Lazy` from `node-core-library`.

Reply:
`Import.lazy` might not make sense here.  It is a perf optimization that loads things on demand the first time they are accessed. Whereas the problem being solved here is to /only/ load the NPM package in a specific code path, because the behavior will be wrong if it is accidentally loaded under some other condition.

I would actually suggest to take it a step further and validate that the `WEBPACK_DEV_SERVER` is undefined before and defined after.

Reply:
We know it'll be defined in the serve case, but we can print a warning if it's somehow become defined in the non-serve case. I'll push a change to do that.